Provider/src/FirebirdSql.Data.FirebirdClient/Client/Managed/Version10/GdsBlob.cs

```diff
@@ -119,40 +119,28 @@ protected override byte[] GetSegment()
 
 				RblRemoveValue(IscCodes.RBL_segment);
 				if (response.ObjectHandle == 1)
-				{
 					RblAddValue(IscCodes.RBL_segment);
-				}
+
```

Comment:
Please keep the brackets (across the whole PR).

Reply:
Added back